nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/controller/StandardFlowServiceTest.java

```diff
@@ -111,13 +112,16 @@ public void testLoadWithFlow() throws IOException {
         String expectedFlow = new String(flowBytes).trim();
         String actualFlow = new String(baos.toByteArray()).trim();
 
-        Assert.assertEquals(expectedFlow, actualFlow);
+        Assertions.assertEquals(expectedFlow, actualFlow);
```

Comment:
This is a good opportunity to replace `Assertions.assertEquals` with the static import of `assertEquals`:
```suggestion
        assertEquals(expectedFlow, actualFlow);
```

Reply:
I can't do that as there are overloaded assertEqual methods in that class.

Reply:
Thanks for the note, I missed that detail on the review. In light of that, the replacement approach currently implemented looks good for now.